<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWDiaballik</class>
 <widget class="QMainWindow" name="MWDiaballik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MWDiaballik</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbPlayer">
      <property name="text">
       <string>Player: GOLD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="selected">
      <property name="text">
       <string>Selected :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>anti-jeu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>swap</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="action_Quitter"/>
   </widget>
   <addaction name="menuOption"/>
  </widget>
  <action name="action_Nouveau">
   <property name="text">
    <string>&amp;Nouveau</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MWDiaballik</receiver>
   <slot>swap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MWDiaballik</receiver>
   <slot>fairPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>swap()</slot>
  <slot>fairPlay()</slot>
 </slots>
</ui>
